homeassistant/components/media_player/horizon.py

```diff
@@ -92,9 +92,11 @@ def supported_features(self):
     @util.Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
     def update(self):
         """Update State using the media server running on the Horizon."""
-        if self._client.is_powered_on():
-            self._state = STATE_PLAYING
-        else:
+        try:
+            self._state = (STATE_PLAYING
+                           if self._client.is_powered_on()
```

Comment:
Please don't use inline if statement, they are not very readable

Reply:
okay ;)